apps/darts-modernisation/darts-api/image-repo.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImageRepository
+metadata:
+  name: lgy-iac-web
```

Comment:
what is this? lgy-iac-web

did you use the scripts to create these files?

Reply:
sorry will update, we weren't able to run the script for all of them encountered a few issues, so used the dts-legacy as a template for what I couldn't get working